Packs/ExabeamSecurityOperationsPlatform/Integrations/ExabeamSecOpsPlatform/ExabeamSecOpsPlatform.py

```diff
@@ -0,0 +1,290 @@
+
+    Args:
+        input_str (str): The input string to be transformed. It should be in the format "key:value".
+
+    Returns:
+        str: The transformed string where the value part is converted to lowercase if it's "true" or "false",
+        otherwise it's enclosed in double quotes.
+    """
+    key, value = input_str.split(':', 1)
+    if value.lower() in ['true', 'false']:
+        return f'{key}:{value.lower()}'
+    else:
+        return f'{key}:"{value}"'
+
+
+def process_string(input_str: str) -> str:
+    """
+    Process the input string by splitting it based on logical operators and transforming each part.
+
+    Args:
+        input_str: The input string to be processed. It may contain logical operators such as 'AND', 'OR', 'NOT', 'TO'.
+
+    Returns:
+        str: The processed string where each part is transformed using the transform_string function.
+    """
+    logical_operators = ['AND', 'OR', 'NOT', 'TO']
+    transformed_parts = []
+    start_index = 0
+
+    for end_index, _char in enumerate(input_str):
+        if any(op in input_str[start_index:end_index] for op in logical_operators):
+            part = input_str[start_index:end_index].strip()
+            if part:
+                operator = next((op for op in logical_operators if op in part), None)
+                if operator:
+                    part = part.replace(operator, "").strip()
+                    transformed_parts.append(transform_string(part))
+                    transformed_parts.append(operator)
+            start_index = end_index + 1
+
+    if start_index < len(input_str):
+        remaining_part = input_str[start_index:].strip()
+        if remaining_part:
+            transformed_parts.append(transform_string(remaining_part))
+
+    return ' '.join(transformed_parts)
+
+
+def _parse_entry(entry: dict):
+    """
+    Parse a single entry from the API response to a dictionary.
+    Args:
+        entry: The entry from the API response.
+    Returns:
+        dict: The parsed entry dictionary.
+    """
+    parsed = {
+        "Id": entry.get("id"),
+        "Raw Log Ids": entry.get("rawLogIds"),
+        "Tier": entry.get("tier"),
+        "Is Parsed": entry.get("parsed"),
+        "Raw Logs": entry.get("rawLogs")
+    }
+    final = remove_empty_elements(parsed)
+    return final if final else None
+
+
+def get_limit(args: dict) -> int:
+    """
+    Get the limit value specified in the arguments.
+
+    Args:
+        args: A dictionary containing the 'limit' argument.
+
+    Returns:
+        int: The limit value if specified and less than or equal to 3000; otherwise, returns 3000 as the maximum limit.
+        If the 'limit' argument is not present in the dictionary or is None, returns 50 as the default limit.
+    """
+    if limit := args.get('limit'):
+        return min(int(limit), 3000)
+
+    return 50
+
+
+def error_fixes(error: str):
+    new_error = ""
+    if error == 'not enough values to unpack (expected 2, got 1)':
```

Comment:
Will it always be with this numbers 2 and 1 i think its better to remove the number part.?